app/scripts/kunden/detail/kundendetail.html

```diff
@@ -299,6 +299,14 @@ <h3 class="panel-title">
                   <input type="text" class="form-control" name="iban" ng-model="kunde.kontoDaten.iban" placeholder="{{'IBAN'|translate}}" ng-iban ng-required="kunde.paymentType==='DirectDebit'">
                 </div>
               </div>
+              <div class="row form-group">
+                <div class="col-md-4">
+                  <label for="bic" translate>BIC</label>
+                </div>
+                <div class="col-md-8">
+                  <input type="text" class="form-control" name="bic" ng-model="kunde.kontoDaten.bic" placeholder="{{'BIC'|translate}}" maxlength=11>
+                </div>
+              </div>
```

Comment:
Above, for the IBAN wie have `ng-required="kunde.paymentType==='DirectDebit'"`. Isn't this true for the BIC as well?

Reply:
no, not mandatory for the client but mandatory for the initiative (still wondering why).

Reply:
ok